src/explorer.ts

```diff
@@ -183,11 +198,13 @@ class KubernetesResourceFolder extends KubernetesFolder {
             host.showErrorMessage(childrenLines.error[0]);
             return [new DummyObject("Error")];
         }
+        if (this.kind.abbreviation === "pod") {
+            const parsedPods = parsePodLine(childrenLines.result);
+            return parsedPods.map((line) => {
```

Comment:
The range variable is now a pod, not a line.

Reply:
will make this change.